src/jest.setup.ts

```diff
@@ -74,6 +74,13 @@ Object.defineProperty(window, "matchMedia", {
   })),
 });
 
+/**
+ * This is needed for getting XTermJs to work in tests
+ */
```

Comment:
How about we override x-term-js with something that mimics the behaviour but is in our full control?

Reply:
I don't think it is worth it. But maybe